docs/dev/table/connectors/formats/avro.md

```diff
@@ -0,0 +1,206 @@
+        <th class="text-center" style="width: 8%">Required</th>
+        <th class="text-center" style="width: 7%">Default</th>
+        <th class="text-center" style="width: 10%">Type</th>
+        <th class="text-center" style="width: 50%">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>format</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Specify what format to use, here should be 'avro'.</td>
+    </tr>
+    </tbody>
+</table>
+
+Data Type Mapping
+----------------
+
+Currently, the Avro schema is always derived from table schema, explicitly define an Avro schema is not supported yet. So here only lists the conversion from Flink type to Avro type.
+
+### Conversion from Flink type to Avro type
+
+The following table lists the conversion from the supported Flink type to Avro type.
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left">Flink Data Type</th>
+        <th class="text-center">Avro type</th>
+        <th class="text-center">Avro logical type</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>CHAR / VARCHAR / STRING</td>
+      <td>string</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>BOOLEAN</td>
+      <td>boolean</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>BINARY / VARBINARY</td>
+      <td>bytes</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>DECIMAL</td>
+      <td>fixed</td>
+      <td>decimal</td>
+    </tr>
+    <tr>
+      <td>TINYINT</td>
+      <td>int</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>SMALLINT</td>
+      <td>int</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>INT</td>
+      <td>int</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>BIGINT</td>
+      <td>long</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>FLOAT</td>
+      <td>float</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>DOUBLE</td>
+      <td>double</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>DATE</td>
+      <td>int</td>
+      <td>date</td>
+    </tr>
+    <tr>
+      <td>TIME</td>
+      <td>int</td>
+      <td>time-millis</td>
+    </tr>
+    <tr>
+      <td>TIMESTAMP</td>
+      <td>long</td>
+      <td>timestamp-millis</td>
+    </tr>
+    <tr>
+      <td>ARRAY</td>
+      <td>array</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>MAP<br>
+      (key must be string/char/varchar type)</td>
+      <td>map</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>MULTISET<br>
+      (element must be string/char/varchar type)</td>
+      <td>map</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>ROW</td>
+      <td>record</td>
+      <td></td>
+    </tr>
+    </tbody>
+</table>
+
+In addition to the types listed above, Flink supports reading/writing nullable types. Flink maps nullable types to Avro `union(something, null)`, where `something` is any converted Avro type from Flink type.
+
```

Comment:
`where `something` is` -> `where `something` can be`

Reply:
I changed this to 

```
where `something` is the Avro type converted from Flink type.
```